www/docs/en/dev/config_ref/index.md

```diff
@@ -299,8 +299,8 @@ android-targetSdkVersion(integer) <br/> ==Android== | *Default: Dependent on cor
 BackupWebStorage(string) <br/> ==iOS== | *Default: cloud* <br/> Allowed values: none, local, cloud. <br/>   Set to cloud to allow web storage data to backup via iCloud. Set to local to allow only local backups via iTunes sync. Set to none prevent web storage backups.
-CordovaWebViewEngine(string) <br/> ==iOS== | *Default: CDVUIWebViewEngine* <br/>  This sets the WebView engine plugin to be used to render the host app. The plugin must conform to the CDVWebViewEngineProtocol protocol. The 'value' here should match the 'feature' name of the WebView engine plugin that is installed. This preference usually would be set by the WebView engine plugin that is installed, automatically.
-CordovaDefaultWebViewEngine(string) <br/> ==iOS== | *Default: CDVUIWebViewEngine* <br/> As the previous setting, CordovaWebViewEngine, this enables you to override the default fallback WebView with a different plugin. The default WebView uses a private plugin, CDVUIWebViewEngine. You can override this if you need to use a public plugin instead. A use case for this setting is where you need to use something other than the default CDVUIWebViewEngine to render the host app. If CordovaDefaultWebViewEngine and CordovaWebViewEngine is set, CordovaWebViewEngine will be chosen to render the host app for devices running iOS9 and above, while CordovaDefaultWebViewEngine will be the fallback for iOS8. The plugin must conform to the CDVWebViewEngineProtocol protocol. The 'value' here should match the 'feature' name of the WebView engine plugin that is installed.
+CordovaWebViewEngine(string) <br/> ==iOS== | *Default: CDVWebViewEngine* <br/>  This sets the WebView engine plugin to be used to render the host app. The plugin must conform to the CDVWebViewEngineProtocol protocol. The 'value' here should match the 'feature' name of the WebView engine plugin that is installed. This preference usually would be set by the WebView engine plugin that is installed, automatically.
+CordovaDefaultWebViewEngine(string) <br/> ==iOS== | *Default: CDVWebViewEngine* <br/> As the previous setting, CordovaWebViewEngine, this enables you to override the default fallback WebView with a different plugin. The default WebView uses a private plugin, CDVWKWebViewEngine. You can override this if you need to use a public plugin instead. A use case for this setting is where you need to use something other than the default CDVWKWebViewEngine to render the host app. If CordovaDefaultWebViewEngine and CordovaWebViewEngine is set, CordovaWebViewEngine will be chosen to render the host app for devices running iOS9 and above, while CordovaDefaultWebViewEngine will be the fallback for iOS8. The plugin must conform to the CDVWebViewEngineProtocol protocol. The 'value' here should match the 'feature' name of the WebView engine plugin that is installed.
```

Comment:
```suggestion
CordovaDefaultWebViewEngine(string) <br/> ==iOS== | *Default: CDVWebViewEngine* <br/> As the previous setting, CordovaWebViewEngine, this enables you to override the default fallback WebView with a different plugin. The default WebView uses a private plugin, CDVWebViewEngine. You can override this if you need to use a public plugin instead. A use case for this setting is where you need to use something other than the default CDVWebViewEngine to render the host app. If CordovaDefaultWebViewEngine and CordovaWebViewEngine is set, CordovaWebViewEngine will be chosen to render the host app for devices running iOS9 and above, while CordovaDefaultWebViewEngine will be the fallback for iOS8. The plugin must conform to the CDVWebViewEngineProtocol protocol. The 'value' here should match the 'feature' name of the WebView engine plugin that is installed.
```

Reply:
Uhh are these sentences even valid?

> CordovaWebViewEngine will be chosen to render the host app for devices running iOS9 and above, while CordovaDefaultWebViewEngine will be the fallback for iOS8.

iOS support <10 is already gone.

Reply:
Definitely dated.

`CordovaDefaultWebViewEngine` can be removed, and if code still exists in the codebase, it can be cleaned up, because like you said... it's not possible to run cordova-ios on iOS <= 10 anymore.